src/reducers/authEventReducer.js

```diff
@@ -0,0 +1,13 @@
+import * as types from '../actions/actionTypes';
+import initialState from './initialState';
+
+
+export default (state = initialState.events, action) => {
+	switch (action.type) {
+	case types.GET_AUTH_EVENT_SUCCESS:
```

Comment:
What do you think about having the switch and case statements on different levels?